Sources/CardPayments/Models/Vault.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+/// This class is used to configure a CardRequest for vaulting
+/// CustomerID is optional. When set, this property will be used to associate  a payment method with vault of customer with this ID
+/// If customerID is not specified, one will be generated by the PayPal API
+public struct Vault {
+    
+    public let customerID: String?
+
+    public init(customerID: String? = nil) {
```

Comment:
These are still missing docstrings

Reply:
Isn't this it? 
/// This class is used to configure a CardRequest for vaulting
/// CustomerID is optional. When set, this property will be used to associate  a payment method with vault of customer with this ID
/// If customerID is not specified, one will be generated by the PayPal API